### PR TITLE
feat: (DSO-00) Add circleCI orb config to repo

### PR DESCRIPTION
**Pull request by javiercm1410**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    attributes:
+      label: "Describe the solution you'd like"
+      description: "A clear and concise description of what you want to happen. Add any considered drawbacks"
+
+
+  - type: textarea
+    attributes:
+      label: "Other"
+      description: |
+        Anything else you want to share?
```
Comment (b-nicasio) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml:34]: No new line at end of file

File: src/examples/example.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+description: >
```
Comment (b-nicasio) [inline src/examples/example.yml]: I think this is not needed

File: src/executors/default.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+description: >
+  This is a sample executor using Docker and Node. If you want to provide a custom environment in your orb, insert your image here.
+  If you do not require an executor, you can simply delete this directory.
```
Comment (b-nicasio) [inline src/executors/default.yml]: Is this file needed?

File: src/jobs/install.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+description: >
+  Sample description
```
Comment (b-nicasio) [inline src/jobs/install.yml]: ?

Comment (javiercm1410) [reply src/jobs/install.yml]: deleted

File: src/executors/default.yml (hunk 3 above)
Comment (javiercm1410) [reply src/executors/default.yml]: deleted

File: src/examples/example.yml (hunk 2 above)
Comment (javiercm1410) [reply src/examples/example.yml]: deleted
